src/pymovements/gaze/_utils/parsing.py

```diff
@@ -416,11 +424,6 @@ def _pre_process_metadata(metadata: defaultdict[str, Any]) -> dict[str, Any]:
         metadata['version_1'], metadata['version_2'],
     )
 
-    if 'resolution' in metadata:
```

Comment:
as above: instead of removing this completely, rename `resolution` to `DISPLAY_COORDS`.